pgctl/cli.py

```diff
@@ -238,6 +239,10 @@ def __change_state(self, state):
                     for service in self.all_services
                 )
 
+        if self._entering_debug:
+            # Do not post-stop when we are about to start a debug session
+            run_post_stop_hook = False
+
```

Comment:
The reason the current behavior is weird is that `pgctl debug` will result in post-stop being called if its the only service in the playground, whereas it won't call post-stop if there's another service.

IMHO the manner that post-stop is called should be the same whether the playground has 1 or >1 services.

Currently pre-start is called upon every pgctl start, regardless of how many services are already running. They're not consistent